### PR TITLE
Be more gentle to slower IPMI controllers

### PR DESCRIPTION
**Pull request by dirkmueller**
Adding a few more sleeps hopefully makes things more reliable.

### CODE REVIEW

**Comment by aplanas** [comment]
+1

**Comment by vmoravec** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1791,7 +1791,11 @@ function reboot_nodes_via_ipmi()
         for pw in 'cr0wBar!' $extraipmipw ; do
             local ip=$bmc_net.$(($ip4 + $i))
             (ipmitool -H $ip -U root -P $pw lan set 1 defgw ipaddr "${bmc_values[1]}"
-            ipmitool -H $ip -U root -P $pw power on
+            sleep 5
+            if ipmitool -H $ip -U root -P $pw power status | grep -q "is off"; then
+                ipmitool -H $ip -U root -P $pw power on
+                sleep 10
+            fi
             ipmitool -H $ip -U root -P $pw power reset) &
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: actually, when the machine was off, you would not need to send a reset anymore - so this could go into an else.

**Comment by vmoravec** [comment]
Patch is not working as expected, we lost a node on qa3 yesterday evening in spite of having this applied. The problem seems to be somewhere else.

**Comment by aplanas** [comment]
@vmoravec maybe this helps? https://github.com/SUSE-Cloud/automation/commit/baa9ad6f51bc6d32227f8bbeee3b6793b4f4121b

**Comment by vmoravec** [comment]
@aplanas thanks for the link, but I doubt it helped, see this log: https://ci.suse.de/job/cloud-mkphyscloud-qa-scenario-2b/97/console

Strange thing is that the machine got IP 
`23:23:57 + ipmitool -H 10.162.24.76 -U root -P '........' lan set 1 defgw ipaddr 10.162.24.1`
then
`23:24:06 + ipmitool -H 10.162.24.76 -U root -P '........' power status`
and later
`23:24:06 + ipmitool -H 10.162.24.76 -U root -P '........' power reset`
`23:24:06 Chassis Power Control: Reset`
and finaly
`23:24:15 Error: Unable to establish LAN session`

**Pull request merged**